tests/test_control_low.py

```diff
@@ -207,6 +207,67 @@ def test_mqtt(self):
             oldmqtt["keep_alive_interval"],
         )
 
+    # Available from fw version 2.7.1
+    @vcr.use_cassette("tests/cassettes/TestControlInterface.test_color.yaml")
+    def test_color(self):
+        ctr = ControlInterface(self.host)
+
+        res = ctr.set_mode("color")._data
+        if self.isrecording:
+            print("set_mode:", res)
+        else:
+            self.assertEqual(res, {"code": 1000})
+
+        res = ctr.set_led_color_rgb(128, 128, 0)._data
+        if self.isrecording:
+            print("set_led_color_rgb:", res)
+        else:
+            self.assertEqual(res, {"code": 1000})
+
+        res = ctr.get_led_color()._data
+        if self.isrecording:
+            print("get_led_color:", res)
+        else:
+            self.assertEqual(
+                res,
+                {
+                    "hue": 60,
+                    "saturation": 255,
+                    "value": 128,
+                    "red": 128,
+                    "green": 128,
+                    "blue": 0,
+                    "code": 1000,
+                },
+            )
+
+        res = ctr.set_led_color_hsv(180, 255, 255)._data
+        if self.isrecording:
+            print("set_led_color_hsv:", res)
+        else:
+            self.assertEqual(res, {"code": 1000})
+
+        res = ctr.get_mode()._data
+        if self.isrecording:
+            print("get_mode:", res)
+        else:
+            self.assertEqual(
+                res,
+                {
+                    "mode": "color",
+                    "shop_mode": 0,
+                    "color_config": {
+                        "hue": 180,
+                        "saturation": 255,
+                        "value": 255,
+                        "red": 0,
+                        "green": 255,
+                        "blue": 255,
```

Comment:
These values don't correspond to `set_led_color_rgb()` - was this recoded in single run?

Reply:
See reply above.